_includes/video/pb-cp.html

```diff
@@ -3,8 +3,7 @@
 
 <head>
 {% include head--common.html %}
-    <!--production version of prebid.js-->
-    <script async src="//cdn.jsdelivr.net/npm/prebid.js@latest/dist/not-for-prod/prebid.js"></script>
+    {% include prebidjs-non-prod.html %}
 
     <!--brightcove & prebid js code-->
```

Comment:
I don't actually see any brightcove code in this page. Maybe this page should be removed?

Reply:
true that